src/cascadia/TerminalApp/AppActionHandlers.cpp

```diff
@@ -1264,6 +1264,54 @@ namespace winrt::TerminalApp::implementation
         }
     }
 
+    void TerminalPage::_HandleSaveTask(const IInspectable& /*sender*/,
+                                       const ActionEventArgs& args)
+    {
+        if (Feature_SaveTask::IsEnabled())

```

Comment:
IMO I'd just do an early-return if the feature _isn't_ enabled, but I'm not sure if that messes with the compiler's ability to just optimize the entire rest of the function out

Reply:
Updated to early return.  (Is there a way to tell if the compiler will struggle with this?)